src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/FuzzyTests.cs

```diff
@@ -13,7 +13,7 @@
 using System.Diagnostics.Tracing;
 #endif
 using Xunit;
-#if USE_ETW // TODO: Enable when TraceEvent is available on CoreCLR. GitHub issue #4864.
+#if USE_ETW
```

Comment:
Do we need a compilation constant for this?  If the issue is just whether to run certain tests on Windows, can we use `[PlatformSpecific(TestPlatforms.Windows)]`?

Reply:
We may be able to get rid of this, but I'm not 100% sure.  Filed #27116.